pkg/config/validation/validation_test.go

```diff
@@ -5897,6 +5897,54 @@ func TestValidateLocalityLbSetting(t *testing.T) {
 			},
 			valid: false,
 		},
+		{
+			name: "invalid failover src contain '*' wildcard",
+			in: &networking.LocalityLoadBalancerSetting{
+				Failover: []*networking.LocalityLoadBalancerSetting_Failover{
+					{
+						From: "*",
+						To:   "region2",
```

Comment:
Why is this invalid now?

Reply:
From the source code analysis: 
The `failoverSetting.From`  only matching the `locality.Region`, and when matching without using the canonical matching, so thai this invalid now.
```go
		// region not match, apply failover settings when specified
		// update localityLbEndpoints' priority to 4 if failover not match
		if priority == 3 {
			for _, failoverSetting := range failover {
				if failoverSetting.From == locality.Region {
					if localityEndpoint.Locality == nil || localityEndpoint.Locality.Region != failoverSetting.To {
						priority = 4
					}
					break
				}
			}
		}
```

Reply:
in my case,i have a lot of regions and zones in a single cluster,  i only want to failover a special region/zone to another region/zone in another cluster ,i use workloadentry, if do not allow '/', how let it work in my case?

Reply:
Why would you need failover from region1.zone1 to region2.zone2, rather than first failover to those ones in same region, like(region1.zone2, region1.zone3)